pkg/scaling/cache/scalers_cache_test.go

```diff
@@ -21,41 +21,45 @@ import (
 func TestTargetAverageValue(t *testing.T) {
 	// count = 0
 	specs := []v2beta2.MetricSpec{}
+	metricName := "queueLength"
```

Comment:
can we add another test case with a different metricName? Just to be sure.

Reply:
sure, doing

Reply:
I'd say that in this case, the name isn't important because it's not asserted anywhere, but I have changed it to `s0-messageCount`, just in case

Reply:
sorry, I don't get your point in that case... we don't check the metric name at any place in that test. I have added the name there just to have it as a parameter because `createMetricSpec` needs it.
I mean, adding another test case with different value for metric name doesn't make sense.